src/styles/Nav.scss

```diff
@@ -0,0 +1,129 @@
+.selected {
+  color: #0275b1 !important;
+}
+.icons > a,
+.user > a,
+.log-out > a {
+  text-decoration: none;
+  color: #181818;
+  &:hover,
+  &:focus,
+  &:active {
+    color: #0275b1;
+  }
+
+  & > p {
+    font-family: Euclid Circular B;
```

Comment:
Should we remove this for now since we don't have the fonts installed in here yet anyway. There is another instance of it below as well.